docs/reference/contributing/guidelines/guidelines.md

```diff
@@ -46,6 +46,10 @@ If so, you can request to become an organization member, but you must first ensu
 - Arm staff must have their Name, Company (Arm) and Location publicly visible.
 - All others should have their Name and Company visible. Including your Location helps us interpret response times according to time zones.
 
+### Leaving ARMmbed organization
+
+Best practices for leaving the organization are documented on [Github help](https://help.github.com/en/articles/best-practices-for-leaving-your-company). Please follow these best practices once you leave the organization.
+
```

Comment:
I would rewrite this to say something like:

When leaving the company please follow the following best practice: <link>

Reply:
fixed